### PR TITLE
fix verdepcheck

### PR DESCRIPTION
**Pull request by pawelru**

### CODE REVIEW

**Comment by pawelru** [comment]
Currently, both "minimal" and "release" strategies fails the test with the following:
```
══ Failed tests ════════════════════════════════════════════════════════════════
── Failure ('test-paginate_listing.R:231'): paginate_to_mpfs works with wrapping on keycols ──
sapply(...) (`actual`) not equal to rep(5, 5) (`expected`).

  `actual`: 4 6 4 6 5
`expected`: 5 5 5 5 5
── Failure ('test-paginate_listing.R:280'): paginate_to_mpfs works with wrapping on keycols when doing horizontal pagination ──
sapply(...) (`actual`) not equal to rep(5, 10) (`expected`).

  `actual`: 4 4 6 6 4 4 6 6 5 5
`expected`: 5 5 5 5 5 5 5 5 5 5

[ FAIL 2 | WARN 0 | SKIP 11 | PASS 73 ]
Error: Test failures
```

The "maximal" works fine.

For me, this looks very much `formatters`-related. If the development version is used (as in "maximal" strategy) - everything is fine. I ran the test locally which confirms my thinking. However, I am not fully convinced as I cannot relate this to a recent changes in `formatters` but I am definitely missing some knowledge here. Calling @insightsengineering/nest-sme for help on this (identical to the similar PR in `rtables`).

File: .github/workflows/scheduled.yaml [hunk 1]
```diff
@@ -2,6 +2,9 @@
 name: Scheduled 🕰️
 
 on:
+  push:
+    branches:
+      - fix_verdepcheck
```
Comment (pawelru) [inline .github/workflows/scheduled.yaml:7]: to be removed

**Comment by Melkiades** [comment]
> Currently, both "minimal" and "release" strategies fails the test with the following:
> 
> ```
> ══ Failed tests ════════════════════════════════════════════════════════════════
> ── Failure ('test-paginate_listing.R:231'): paginate_to_mpfs works with wrapping on keycols ──
> sapply(...) (`actual`) not equal to rep(5, 5) (`expected`).
> 
>   `actual`: 4 6 4 6 5
> `expected`: 5 5 5 5 5
> ── Failure ('test-paginate_listing.R:280'): paginate_to_mpfs works with wrapping on keycols when doing horizontal pagination ──
> sapply(...) (`actual`) not equal to rep(5, 10) (`expected`).
> 
>   `actual`: 4 4 6 6 4 4 6 6 5 5
> `expected`: 5 5 5 5 5 5 5 5 5 5
> 
> [ FAIL 2 | WARN 0 | SKIP 11 | PASS 73 ]
> Error: Test failures
> ```
> 
> The "maximal" works fine.
> 
> For me, this looks very much `formatters`-related. If the development version is used (as in "maximal" strategy) - everything is fine. I ran the test locally which confirms my thinking. However, I am not fully convinced as I cannot relate this to a recent changes in `formatters` but I am definitely missing some knowledge here. Calling @insightsengineering/nest-sme for help on this (identical to the similar PR in `rtables`).

This is related to the pagination + wrapping fix I did recently. I am happy to see that the tests I added are spotting this instantly. I do not know what is the cause though

**Review by Melkiades — APPROVED** [review]
Lgtm! Thanks Pawel :)

**Pull request merged**